package.json

```diff
@@ -36,19 +36,19 @@
   "author": "Aldwin Vlasblom <aldwin@avaq.it> (https://github.com/Avaq)",
   "license": "MIT",
   "dependencies": {
-    "sanctuary-type-identifiers": "^2.0.0"
+    "sanctuary-type-identifiers": "^3.0.0"
   },
   "peerDependencies": {
-    "fluture": ">=12.0.0-beta.5 <13.0.0",
-    "sanctuary-def": ">=0.20.0 <0.21.0"
+    "fluture": ">=12.0.0-beta.6 <13.0.0",
+    "sanctuary-def": ">=0.20.0 <0.22.0"
```

Comment:
Can one use `sanctuary-def@0.20.0` with `sanctuary-type-identifiers@3.0.0`?

Reply:
No, but that's not what would happen. NPM would install two versions of sanctuary-type-identifiers. One used by fluture-sanctuary-types, and one by sanctuary-def.

Since fluture-sanctuary-types doesn't use sanctuary-type-identifiers on any types except those provided by Fluture (with the exception of in the tests), it doesn't matter which version of sanctuary-def is being used.